crates/ruma-common/tests/events/room_message.rs

```diff
@@ -434,6 +435,60 @@ fn content_deserialization_failure() {
     assert_matches!(from_json_value::<RoomMessageEventContent>(json_data), Err(_));
 }
 
+#[test]
+fn escape_tags_in_plain_reply_body() {
+    let first_message = OriginalRoomMessageEvent {
+        content: RoomMessageEventContent::text_plain("Usage: cp <source> <destination>"),
+        event_id: event_id!("$143273582443PhrSn:example.org").to_owned(),
+        origin_server_ts: MilliSecondsSinceUnixEpoch(uint!(10_000)),
+        room_id: room_id!("!testroomid:example.org").to_owned(),
+        sender: user_id!("@user:example.org").to_owned(),
+        unsigned: MessageLikeUnsigned::default(),
+    };
+    let second_message = OriginalRoomMessageEvent {
+        content: RoomMessageEventContent::text_plain("Usage: rm <path>")
+            .make_reply_to(&first_message, ForwardThread::Yes),
+        event_id: event_id!("$143273582443PhrSn:example.org").to_owned(),
+        origin_server_ts: MilliSecondsSinceUnixEpoch(uint!(10_000)),
+        room_id: room_id!("!testroomid:example.org").to_owned(),
+        sender: user_id!("@user:example.org").to_owned(),
+        unsigned: MessageLikeUnsigned::default(),
+    };
```

Comment:
Thanks for the new test!

There's no need to construct full events though, since we only care about the content.

Reply:
I just stole the one from right below, since it was the only one that used `make_reply_to()`, but I'll look into minimizing it.

Reply:
I've removed the generation of the second event; the first one is still necessary in order to create a reply to it.